src/Backend/Modules/ContentBlocks/Actions/ContentBlockEdit.php

```diff
@@ -0,0 +1,131 @@
+<?php
+
+namespace Backend\Modules\ContentBlocks\Actions;
+
+/*
+ * This file is part of Fork CMS.
+ *
+ * For the full copyright and license information, please view the license
+ * file that was distributed with this source code.
+ */
+
+use Backend\Core\Engine\Authentication;
+use Backend\Core\Engine\Base\ActionEdit as BackendBaseActionEdit;
+use Backend\Core\Engine\Model as BackendModel;
+use Backend\Modules\ContentBlocks\Domain\ContentBlock\Command\UpdateContentBlock;
+use Backend\Modules\ContentBlocks\Domain\ContentBlock\ContentBlock;
+use Backend\Modules\ContentBlocks\Domain\ContentBlock\Event\ContentBlockUpdated;
+use Backend\Modules\ContentBlocks\Domain\ContentBlock\ContentBlockRepository;
+use Backend\Modules\ContentBlocks\Domain\ContentBlock\ContentBlockType;
+use Backend\Modules\ContentBlocks\Domain\ContentBlock\ContentBlockRevisionDataGrid;
+use Backend\Core\Language\Locale;
+use Backend\Modules\ContentBlocks\Domain\ContentBlock\Exception\ContentBlockNotFound;
+use Symfony\Component\Form\Form;
+
+/**
+ * This is the edit-action, it will display a form to edit an existing item
+ */
+class ContentBlockEdit extends BackendBaseActionEdit
+{
+    public function execute(): void
+    {
+        parent::execute();
+
+        /** @var ContentBlock $contentBlock */
+        $contentBlock = $this->getContentBlock();
+
+        /** @var Form $form */
+        $form = $this->getForm($contentBlock);
+
+        if (!$form->isValid()) {
```

Comment:
checking if a form is valid before checking if a form is submitted is deprecated in symfony 3.3 and removed in symfony 4, since we will need to upgrade for fork 5 this will need to be changed here as well